code/datums/gamemodes/crash.dm

```diff
@@ -187,7 +187,7 @@
 	SIGNAL_HANDLER
 	switch(new_xeno.tier)
 		if(XENO_TIER_ONE)
-			new_xeno.upgrade_stored = max(new_xeno.upgrade_stored, TIER_ONE_MATURE_THRESHOLD)
+			new_xeno.upgrade_stored = min(new_xeno.upgrade_stored, TIER_ONE_THRESHOLD)
```

Comment:
Don't think this proc is necessary at all anymore considering this is meant to give extra mat on first evo and you're removing mat.